cli/parse.go

```diff
@@ -85,6 +102,23 @@ func (st *parseState) peek() string {
 	return st.cmdline[st.i]
 }
 
+func setOpts(kv kv, kvType reflect.Kind, opts cmds.OptMap) error {
+
+	if _, exists := opts[kv.Key]; exists && kvType != cmds.Strings {
+		return fmt.Errorf("multiple values for option %q", kv.Key)
+	}
```

Comment:
We can simplify this by moving the "exists" check into the else statement of the conditional below. Then we don't need the `kvType != cmd.Strings` check.

That is:

```go
if kvType == cmd.Strings {
...
} else if _, exists := opts[kv.Key]; !exists {
    ...
} else {
    return err
}
```